data/item_types.json

```diff
@@ -24,6 +26,17 @@
     "organisation": {
       "$ref": "https://ils.rero.ch/api/organisations/1"
     },
+    "negative_availability": false,
+    "circulation_information": [
+      {
+        "language": "eng",
+        "label": "On-site consultation"
+      },
+      {
+        "language": "fre",
+        "label": "Consultation sur place"
+      }
```

Comment:
In the interface we also have Italian and German. Do we have to add Italian and German here?

Reply:
We could, add `de` and `it` translation into data but it's not required.
The UUI will try to use the correct language label, if none label are defined for the UI language, then the itty_name will be used as default value.